File: Herencia/Ejercicio3/Java/Docente.java
package Herencia.Ejercicio3.Java;

public class Docente extends Persona {
    private String nit, profesion, especialidad, sexo;

    public Docente(String nom, String ap, int ci, int cel, int d, int m, int a, String nit, String profesion, String especialidad, String sexo) {
        super(nom, ap, ci, cel, d, m, a);
        this.nit = nit;
        this.profesion = profesion;
        this.especialidad = especialidad;
        this.sexo = sexo;

}
    void mostrar() {
        super.mostrar();
        System.out.println("NIT: " + nit);
        System.out.println("Profesión: " + profesion); 
        System.out.println("Especialidad: " + especialidad); 
        System.out.println("Sexo:"+ sexo);
    }

    String getProfesion() { return profesion; } 
    String getSexo() { return sexo; }
}

File: Herencia/Ejercicio3/Python/Docente.py

from Persona import Persona
class Docente(Persona):
    def __init__(self, nombre, apellido, ci, celular, d, m, a, nit, profe, espe, sexo):
        super().__init__(nombre, apellido, ci, celular, d, m, a)
        self.nit = nit
        self.profesion = profe
        self.especialidad = espe
        self.sexo = sexo
        
    def mostrar(self):
        super().mostrar()
        print(f"NIT: {self.nit}")
        print(f"Profesión: {self.profesion}")
        print(f"Especialidad: {self.especialidad}")
        print(f"Sexo: {self.sexo}")
        
    def getProfesion(self):  return self.profesion
    def getSexo(self):  return self.sexo

